ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js

```diff
@@ -53,15 +53,17 @@ function validateContactFrom(e) {
     }
 
    
-	const ErrorMsgElement = document.getElementById('cmp-contactus-errormsg');
-    if(validateRecaptcha() == false) {
-        ErrorMsgElement.style.visibility = "visible";
-        ErrorMsgElement.innerHTML = '<span>please verify you are human!<span>';
-         isValid = false;
-        return false;
-    } else {
-        ErrorMsgElement.style.visibility = "hidden";
-        ErrorMsgElement.innerHTML = '';
+    const ErrorMsgElement = document.getElementById('cmp-contactus-errormsg');
+    if(value){
```

Comment:
@nitheshkumar-hcl : what value is this ? please add a relevant variable name

Reply:
also put variable declaration in contactus.js

Reply:
Variable Name Changed as per the review comments.variable declaration for contact us.js is provided in recaptcha.js